### ISSUE TITLE
Permission denied

### ISSUE THREAD

**Issue by neidiom** [systemli/ansible-role-roundcube#27]
I am getting permission issues when I try to access roundcube via nginx

````
*9 stat() "/var/www/roundcube.mydomain.com/www/public_html/index.php" failed (13: Permission denied),
````

this is the fix

````
chmod +x /var/www/roundcube.mydomain.com/www/public_html
chmod +x /var/www/roundcube.mydomain.com/www/
chmod +x /var/www/roundcube.mydomain.com/
````

**Comment by neidiom** [comment]
fixed at https://github.com/systemli/ansible-role-roundcube/pull/28

**Comment by doobry-systemli** [comment]
This seems like a configuration issue on your side to me. The role itself doesn't deploy a directory called `public_html`. What do you have set as `roundcube_user_home` and `roundcube_path`?

**Comment by neidiom** [comment]
There is a public_html directory at roundcube, see https://github.com/roundcube/roundcubemail/tree/master/public_html

**Comment by neidiom** [comment]
I have improved the PR.

**Comment by doobry-systemli** [comment]
Fixed by #28
